recipes/hg-evolve/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hg-evolve" %}
+{% set version = "8.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7ef633c1ab9ef4ef9da8c74165a7b12daa32a38d6b3de9324ba4ce2b42ea126
+
+build:
+  number: 0
+  skip: True  # [py3k]
```

Comment:
Skip can't be used with `noarch` packages but you can apply the Python version constraint in a different way.
```suggestion
```